### ISSUE TITLE
VarTypeHint doesn't check function arguments

### ISSUE THREAD

**Issue by DalekCraft2** [vshaxe/vscode-checkstyle#19]
VSCode v1.72.2, Haxe Checkstyle v1.8.1

I have the `VarTypeHint` option set to `enforce_all`, and it does detect lack of type hints most of the time, but it doesn't seem to notice when the arguments of a function lack type hints.
If I had this:
```haxe
public function test(anArg, anotherArg, fooBar):Void {}
```
it would not show any warnings for the missing types.

Is this a bug with this extension or with Haxe Checkstyle itself? I have more things to report and I don't want to send a bunch of bug reports in the wrong repository.

**Comment by AlexHaxe** [comment]
`VarTypeHint` only checks for variables (`var` and `final`) not function parameters. as far as I'm aware there is no test to enforce type hints for function parameters.

and that would be a checkstyle issue since it concerns checks, their scope and their configuration. 
it would be an extension issue, if one of the quick fix code actions worked incorrectly or you had a proposal for enhancing or adding a code action.
